ovs/v1/ovs.yaml

```diff
@@ -590,13 +348,6 @@ components:
       in: path
       name: eventID
       schema:
-        $ref: 'https://api.swaggerhub.com/domains/dcsaorg/DCSA_EVENT_DOMAIN/1.0.0#/components/schemas/eventID'
-      required: true
-      description: The ID of the timestampID to filter by
-    subscriptionID:  
-      in: path
-      name: subscriptionID
-      schema:
-        $ref: 'https://api.swaggerhub.com/domains/dcsaorg/DCSA_OVS_DOMAIN/1.0.0#/components/schemas/subscriptionID'
+        $ref: 'https://api.swaggerhub.com/domains/dcsaorg/EVENT_DOMAIN/1.0.0#/components/schemas/eventID'
       required: true
-      description: The ID of the subscription to receive
\ No newline at end of file
+      description: The ID of the timestampID to filter by
```

Comment:
@holmHin are you sure about this change? I would expect this to be the description of the eventID? and not a timestampID?